CHANGES.txt

```diff
@@ -18,41 +18,5 @@ Breaking Changes
 Changes
 =======
 
- - Updated Elasticsearch to v5.6.8
```

Comment:
Why are the changes not in this file?

Reply:
@emilywoods How do you mean? The CHANGES.txt file just tracks unreleased breaking changes, regular changes and bug fixes. When we release, we just move the unreleased changes here to a released changes document. Usually hotfix releases don't contain changes, but since this ES upgrade was just from a hotfix update, its accetable

Reply:
Ah ok - I misunderstood how `changes.txt` was being used. Thanks :)